shared/src/main/scala/mlscript/Typer.scala

```diff
@@ -49,6 +49,7 @@ class Typer(var dbg: Boolean, var verbose: Bool, var explainErrors: Bool)
     def nest: Ctx = copy(Some(this), MutMap.empty, MutMap.empty)
     def nextLevel: Ctx = copy(lvl = lvl + 1)
     private val abcCache: MutMap[Str, Set[Var]] = MutMap.empty
+    // USE THIS!!!
```

Comment:
You should not make me have to clean after yourself... Please properly review your diff before asking for a review! 😠